charts/tidb-operator/values.yaml

```diff
@@ -217,14 +217,18 @@ admissionWebhook:
     ## apiservice config
     ## refer to https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#contacting-the-extension-apiserver
     insecureSkipTLSVerify: true
-    ## The key and cert for `tidb-admission-webook.<Release Namespace>` Service.
+    ## The tls key and cert for `tidb-admission-webook.<Release Namespace>.svc` Service in Secret.
     ## If insecureSkipTLSVerify is true, this would be ignored.
-    cert: ""
-    key: ""
+    ## You can create the tls secret by:
+    ## kubectl create secret tls tidb-admission-webhook-certs  --cert=<path-to-cert> --key=<path-to-key> --namespace <release-namespace>
+    tlsSecret: ""
+    ## The caBundle for the webhook apiservice, you could get it by the secret you created previously:
+    ## kubectl get secret tidb-admission-webhook-certs -o=jsonpath='{.data.tls\.crt}'
+    caBundle: ""
```

Comment:
There is another caBundle in L234, we need both of them?

Reply:
yes, these are for different cabundle setting.